pkg/api/types.go

```diff
@@ -311,3 +331,48 @@ func (p *PullPolicy) Set(v string) error {
 	}
 	return nil
 }
+
+// IsInvalidFilename verifies if the provided filename contains malicious
+// characters.
+func IsInvalidFilename(name string) bool {
+	return !strings.ContainsAny(name, InvalidFilenameCharacters)
+}
+
+// Set implements the Set() function of pflags.Value interface.
+// This function parses the string that contains source:destination pair.
+// When the destination is not specified, the source get copied into current
+// working directory in container.
+func (il *InjectionList) Set(value string) error {
+	mount := strings.Split(value, ":")
+	switch len(mount) {
+	case 0:
+		return fmt.Errorf("invalid format, must be source:destination")
+	case 1:
+		mount = append(mount, "")
```

Comment:
`fallthrough` is a DRY solution, otherwise:

```go
case 1:
    mount[0] = strings.Trim(mount[0], `"'`)
    mount = append(mount, "")
case 2:
...
```